Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml

```diff
@@ -229,7 +229,7 @@
       fly-by: *flybyfixture
   - type: Ammo
   - type: StaminaDamageOnCollide
-    damage: 30
+    damage: 33
```

Comment:
Not sure what changing from 30 to 33 actually does, but I guess it doesn't hurt.